person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
```

Comment:
- [optional] Since it is a convention for many Ruby programmer, to bring the  public methods at the top and private at the bottom, I think it will be nice if you can do the same in order to eliminate confusion when working on your codebase :+1: